<launch>
    <arg name = "car_name" default = "car_4"/>
    <arg name = "trajectory_name" default = "raceline"/>
    <arg name = "lookahead_distance" default = "raceline"/>
    <arg name = "wheelbase" default = "0.325"/>
    <include file="$(find particle_filter)/launch/slam.launch" />    
    <include file="$(find move_base)/launch/move_base.launch" >
        <param name="listen_offboard" value="true" />
    </include>

    <node pkg = "f1tenth_purepursuit" type = "pure_pursuit.py" name = "pure_pursuit" >
        <param name = "car_name" value = "$(arg car_name)"/>
        <param name = "trajectory_name" value = "$(arg trajectory_name)"/>
        <param name = "lookahead_distance" value = "$(arg lookahead_distance)"/>
        <param name = "wheelbase" value = "$(arg wheelbase)"/>
    </node>
</launch>